ext/node/Cargo.toml

```diff
@@ -25,8 +25,6 @@ deno_fetch.workspace = true
 deno_fs.workspace = true
 deno_media_type.workspace = true
 deno_net.workspace = true
-deno_npm.workspace = true
-deno_semver.workspace = true
```

Comment:
This change was done because we want ext/node to work solely on paths so that byonm can occur in the future.